stable/spring-cloud-data-flow/templates/skipper-deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: extensions/v1beta1
```

Comment:
Please update to `apps/v1` and add a `spec.selector`.

Reply:
updated